README.md

```diff
@@ -65,58 +50,146 @@ To clean up files generated from the build, you can simply run:
+$ sls deploy list
+Serverless:
+-----------
+Name: myFunctionApp-t1561479533
+Timestamp: 1561479533
+Datetime: 2019-06-25T16:18:53+00:00
+-----------
+Name: myFunctionApp-t1561479506
+Timestamp: 1561479506
+Datetime: 2019-06-25T16:18:26+00:00
+-----------
+Name: myFunctionApp-t1561479444
+Timestamp: 1561479444
+Datetime: 2019-06-25T16:17:24+00:00
+-----------
+
+# Rollback Function App to timestamp
+$ sls rollback -t 1561479506
+```
+
+This will update the app code and infrastructure to the selected previous deployment.
+
+For more details, check out our [rollback docs](docs/ROLLBACK.md).
+
+### Deleting Your Function App
 
-The getting started walkthrough illustrates the interactive login experience, which is recommended for most users. However, if you'd prefer to create an Azure ["service principal"](https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/Authentication.md#2-azure-cli) yourself, you can indicate that this plugin should use its credentials instead, by setting the following environment variables:
+If at any point you no longer need your service, you can run the following command to delete the resource group containing your Azure Function App and other depoloyed resources using:
 
-**Bash**
 ```bash
-export azureSubId='<subscriptionId>'
-export azureServicePrincipalTenantId='<tenantId>'
-export azureServicePrincipalClientId='<servicePrincipalName>'
-export azureServicePrincipalPassword='<password>'
+$ sls remove
 ```
 
-**Powershell**
-```powershell
-$env:azureSubId='<subscriptionId>'
-$env:azureServicePrincipalTenantId='<tenantId>'
-$env:azureServicePrincipalClientId='<servicePrincipalName>'
-$env:azureServicePrincipalPassword='<password>'
+### Creating or removing Azure Functions
+
+To create a new Azure Function within your function app, run the following command from within your app's directory:
+
+```bash
+# -n or --name for name of new function
+$ sls func add -n {functionName}
 ```
 
+This will create a new handler file at the root of your project with the title `{functionName}.js`. It will also update `serverless.yml` to contain the new function.
+
+To remove an existing Azure Function from your function app, run the following command from within your app's directory:
+
+```bash
+# -n or --name for name of function to remove
+$ sls func remove -n {functionName}
+```
+
+This will remove the `{functionName}.js` handler and remove the function from `serverless.yml`
+
+*Note: Add & remove currently only support HTTP triggered functions. For other triggers, you will need to update `serverless.yml` manually
+
+### Advanced Authentication
+
+The getting started walkthrough illustrates the interactive login experience, which is recommended when getting started. However, for more robust use, a [service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals) is recommended for authentication.
+
+##### Creating a Service Principal
```

Comment:
can you add reference to use the script I wrote?

Reply:
We should talk about where that should live. I don't know if it's in the quickstart, just because we expose that script in the `scripts` section of `package.json`, and it would be kinda awkward for people that were consuming the plugin to drill into `node_modules` to go run the script. If that script were available in the project template, then I think it would make sense to be here. I think that would go in other more detailed dev docs, just because we'd have to have other instructions like installing `jq`. Open to chat about where to put that in the future, but I'll probably leave it out for now